### PR TITLE
fix(jetbrains): link to indexing per IDE

### LINKED ISSUE #2232: [Docs] Incorrect link on JetBrains product pages
### Page URL

https://www.gitpod.io/docs/ides-and-editors/intellij#indexing-using-prebuilds

### User feedback

In JetBrains product pages the `general recommendations` link always to GoLand docs instead of being dynamic for IDE. 

Instead, links should go to:

IntelliJ: https://www.jetbrains.com/help/idea/indexing.html
PhpStorm: https://www.jetbrains.com/help/phpstorm/indexing.html
PyCharm: https://www.jetbrains.com/help/pycharm/indexing.html

---
Internal chat: https://gitpod.slack.com/archives/C02CBKZEVS8/p1655126652923109

### PR DESCRIPTION
**Pull request by andreafalzetti**
## Description
Link to external docs should be per IDE product, at the moment it always send users to the GoLand specific page.

## Related Issue(s)

Fixes #2232

## How to test
1. Visit https://deploy-preview-2234--gitpod-kumquat.netlify.app/docs/ides-and-editors
2. Check each JB IDE product page and verify that the link to `general recommendations` links to the right IDE

## Release Notes

```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2234"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2234"></a>

### CODE REVIEW

File: generators/ide/jb-gen.js [hunk 1]
```diff
@@ -15,18 +15,22 @@ products.set("intellij", {
   productTitle: "IntelliJ IDEA",
   productFullyQualifiedTitle: "IntelliJ IDEA Ultimate",
   productDocCode: "idea",
+  externalDocsIndexing: "https://www.jetbrains.com/help/idea/indexing.html",
```
Comment (akosyakov) [inline generators/ide/jb-gen.js:18]: I think idea was to use `productDocCode`

File: generators/ide/jb-product.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ When you open the project {title} starts indexing to load modules and
 enable the core functionality like code completion and navigation.
 Depending on the size of your project indexing speed can vary significantly.
 
-You can speed up the indexing of a project by applying these [general recommendations](https://www.jetbrains.com/help/go/indexing.html).
```
Comment (akosyakov) [inline generators/ide/jb-product.md:128]: `https://www.jetbrains.com/help/__productDocCode__/indexing.htm` should be enough?

**Comment by gtsiolis** [comment]
Thanks for tackling this, @andreafalzetti! 🏀

File: generators/ide/jb-gen.js (hunk 1 above)
Comment (andreafalzetti) [reply generators/ide/jb-gen.js]: I completely missed that, updated

**Pull request merged**